ROADMAP.md

```diff
@@ -1,16 +1,37 @@
-# KServe 2023 Roadmap
+# KServe 2024-2025 Roadmap
```

Comment:
Would it be a good idea to link each item to an existing/new GitHub issue for deeper discussions on each topic?

Reply:
linked the issues for these items